packages/admin/cms-admin/src/cronJobs/JobLogs.tsx

```diff
@@ -64,22 +65,24 @@ export function JobLogs(props: { jobName: string }) {
     const job = data?.kubernetesJob;
 
     return (
-        <MainContent disablePadding>
+        <>
             <JobLogsToolbar kubernetesJob={job} />
-            {logs ? (
-                <Box paddingLeft={4}>
-                    <LogsContainer>{logs}</LogsContainer>
-                </Box>
-            ) : (
-                <Box padding={4}>
-                    <Typography>
-                        <FormattedMessage
-                            id="comet.cronJobs.noLogs"
-                            defaultMessage="No logs available. You may consider checking external logging tools if available."
-                        />
-                    </Typography>
-                </Box>
-            )}
-        </MainContent>
+            <MainContent>
```

Comment:
Normally, toolbar and MainContent should be on Page level, but here there is no easy way to set the ToolbarTitle. I also didn't find a good way to set the breadcrumb title, so I left the defaults.

Reply:
You are right for `JobLogs`. I completely forgot about `StackPageTitle`.

However it doesn't work for `JobsGrid` at the moment. We have a setup like this:

```
<StackPage name="jobs" title="Outside">
    {(selectedCronJobName) => (
        <StackSwitch>
            <StackPage name="grid" title="Inside">
```

In this case, the breadcrumb and `ToolbarAutomaticTitleItem ` show "Outside". But `StackPageTitle` overrides "Inside".

In my opinion the behavior of the Stack is wrong here. It should prefer "Inside" over "Outside". I looked into it but couldn't find a quick way to change this.